src/Poseidon/CLI/OptparseApplicativeParsers.hs

```diff
@@ -418,3 +419,16 @@ parseUpgrade = OP.switch (
     OP.long "upgrade" <>  OP.short 'u' <>
     OP.help "overwrite outdated local package versions"
     )
+
+-- PlinkPopNameAsFamily always is the default
+parsePlinkPopMode :: OP.Parser PlinkPopNameMode
+parsePlinkPopMode = parsePlinkPopAsPhenotype <|> parsePlinkPopAsBoth <|> pure PlinkPopNameAsFamily
```

Comment:
I think I would prefer one option with three possible iterations of an argument (e.g. `--plinkPopName asPhenotype | asBoth | asFamily`). It's more flexible in case other options emerge in the future (unlikely, I know, but still), fits to the other options we already have (e.g. `--outFormat`) and, most importantly, is less verbose and repetitive in the CLI documentation.
![image](https://user-images.githubusercontent.com/10789038/217078084-c17f7c49-d0da-4a1a-9520-e744181feab9.png)
But I understand that this is a matter of taste.

Reply:
No I agree, actually. Good point. I'll change this.

Reply:
Done.